libs/types-common-aws/src/Util/Test/SQS.hs

```diff
@@ -54,17 +53,29 @@ data SQSWatcher a = SQSWatcher
 -- the queue has too many things in it before the tests start.
 -- Note that the purgeQueue command is not guaranteed to be instant (can take up to 60 seconds)
 -- Hopefully, the fake-aws implementation used during tests is fast enough.
-watchSQSQueue :: Message a => AWS.Env -> Text -> IO (SQSWatcher a)
-watchSQSQueue env queueUrl = do
+watchSQSQueue :: (Message a) => AWS.Env -> Text -> IO (SQSWatcher a)
+watchSQSQueue env queueName = do
   eventsRef <- newIORef []
-  ensureEmpty
-  process <- async $ recieveLoop eventsRef
+
+  queueUrlRes <- execute env . sendEnv $ SQS.newGetQueueUrl queueName
+  let queueUrl = view SQS.getQueueUrlResponse_queueUrl queueUrlRes
+
+  ensureEmpty queueUrl
+  process <- async $ do
+    loop1 <- async $ recieveLoop queueUrl eventsRef
+    loop2 <- async $ recieveLoop queueUrl eventsRef
+    loop3 <- async $ recieveLoop queueUrl eventsRef
+    loop4 <- async $ recieveLoop queueUrl eventsRef
+    loop5 <- async $ recieveLoop queueUrl eventsRef
+    _ <- Async.waitAny [loop1, loop2, loop3, loop4, loop5]
```

Comment:
Hm, somehow I don't get why you're calling `recieveLoop` five times here with the same parameters? :thinking: 
Maybe, leave a comment to explain this?

Reply:
I added a comment. Basically adding more loop increases the throughput of the consumer making tests not have to wait too long.